blip-tap.md

```diff
@@ -919,149 +919,251 @@ order to clamp down on price volatility exposure.
   until this time.
-
-* `rfq_sig` is a signature over the serialized contents of the message 
+* `rfq_sig` is a signature over the serialized contents of the message.
+* `in_asset_to_btc` is the in-asset to BTC conversion rate represented as a
+  [fixed-point number](#fixed-point-number-type).
+* `out_asset_to_btc` is the out-asset to BTC conversion rate represented as a
+  [fixed-point number](#fixed-point-number-type).
 
 ##### Requirements
 
 The sender:
 
-  - MUST set `rfq_id` to the matching `rfq_id` sent in a prior `tap_rfq` message
+- MUST set `rfq_id` to the matching `rfq_id` sent in a prior `tap_rfq` message.
 
-  - MUST set `accepted_rate_tick` to a value they deem to be an acceptable
-    exchange rate
+- MUST set `expiry` to a future unix timestamp after which the quote is no
+  longer valid.
 
-  - MUST set `expiry` to a future unix timestamp after which the quote is no
-    longer valid
+- MUST set `rfq_sig` to be a BIP-340 schnorr signature over the serialized
+  contents of the message without the `rfq_sig` serialized, using their node
+  public key.
 
-  - MUST set `rfq_sig` to be a BIP-340 schnorr signature over the serialized
-    contents of the message without the `rfq_sig` serialized, using their node
-    public key
+- MUST set the fields `in_asset_to_btc` and `out_asset_to_btc` to appropriate
+  values based on the asset exchange rates.
 
 The recipient:
 
-  - MUST abandon the attempt if `rfq_sig` is invalid
+- MUST abandon the RFQ session if `version` corresponds to an unsupported
+  version.
+
+- MUST abandon the RFQ session if `rfq_sig` is invalid.
 
-  - MUST abandon the attempt if they deem that `accepted_rate_tick` is
-    unreasonable
+- MUST abandon the attempt if they deem that `in_asset_to_btc`
+  or `out_asset_to_btc` is unreasonable.
 
-  - SHOULD no longer attempt to utilize the cleared quote after unix timestamp
-    `expiry`
+- SHOULD no longer attempt to utilize the cleared quote after unix timestamp
+  `expiry`.
 
 #### Rejecting Quotes  (`tap_rfq_reject`) 
 
 In the event that an edge node is unable to satisfy a quote request, then they
 should send `tap_rfq_reject`, identifying the rejected quote ID. A quote might
 be rejected if the channel cannot accommodate the proposed volume, or if the
-edge node is unwilling to carry any HTLCs for that `asset_id`.
+edge node is unwilling to carry any HTLCs for the subject TAP asset(s).
 
-1. type: ?? (`tap_req_accept`)
-2. data:
-     * [`32*byte`:`rfq_id`]
+The `tap_rfq_reject` message has a BOLT 01 wire message type integer of 52886,
+falling within the custom message range. Its TLV stream consists of the
+following TLV fields:
+
+| TLV Type | Field Name | Field Type |
+|----------|------------|------------|
+| 0        | version    | 1*byte     |
+| 1        | rfq_id     | 32*byte    |
 
 where:
 
-* `rfq_id` is the quote ID that they wish to reject
+* `version` is the version of the quote reject message. The current version is
+  `0`.
```

Comment:
Same here re version bump.

Reply:
I don't believe we need to bump the version for the reject message type, as we are updating the BLIP to align with what has already been released in TAPD.

Reply:
I noticed in your RFQ refactor PR that the error code and error message fields have been nested under a new field.

Reply:
Bumped to 1 now.